src/js/components/DataFilters/DataFilters.js

```diff
@@ -100,7 +100,7 @@ export const DataFilters = ({ drop, children, heading, layer, ...rest }) => {
               }))
           : undefined
       }
-      {...(!controlled ? rest : {})}
+      {...(!controlled ? rest : { fill: 'vertical' })}
```

Comment:
Is controlled when the user isn't passing their own DataFilter? it feels like fill: vertical would always be helpful if that enables the overflow scroll.

Reply:
When rendering inline, it allows the caller to decide how to handle it. Do you have a specific scenario you are envisioning?